userspace/ksud/src/module.rs

```diff
@@ -157,8 +151,19 @@ fn switch_cgroups() -> Result<()> {
     switch_cgroup("/acct", pid);
     switch_cgroup("/dev/cg2_bpf", pid);
     switch_cgroup("/sys/fs/cgroup", pid);
-    if getprop("ro.config.per_app_memcg")? != "false" {
-        switch_cgroup("/dev/memcg/apps", pid);
+
+    let per_app_memcg = getprop("ro.config.per_app_memcg");
+    if (per_app_memcg.is_some()
+        && per_app_memcg
+            .clone()
+            .filter(|prop| prop != "0" && prop != "false")
+            .is_some())
+        || (per_app_memcg.is_none()
+            && getprop("ro.config.low_ram")
```

Comment:
`ro.config.low_ram` doesn't always use `/dev/memcg/apps`? Magisk don't have this.

Reply:
`ro.config.per_app_memcg` seems to have higher priority.

Reply:
If it have issue, please make another PR, don't modify behavior in this.